pkg/service/operation/storage/bolt.go

```diff
@@ -1,18 +1,40 @@
 package storage
 
 import (
+	"fmt"
 	"github.com/goccy/go-json"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/tbd54566975/ssi-service/internal/util"
 	"github.com/tbd54566975/ssi-service/pkg/storage"
 	"go.einride.tech/aip/filtering"
+	"strings"
 )
 
 const (
-	namespace = "operation"
+	namespace  = "operation"
+	submission = "submission"
```

Comment:
I'm not sure why we need different buckets?
Why can't all operations live in the operations bucket?

If we do need a submission bucket can we name it operations-submission or something so that we know there is no overlap with presentation bucket

Reply:
> Why can't all operations live in the operations bucket?

They could! I decided to have separate buckets per operation response type because I expect that's going to be way in which they get queried. I imagine that the UI will have the following 2 screens:
* Show me all pending operations related to submissions;
* Show me all pending operations related to credential applications;

The bucket name actually gets resolved into `operation_submission`. see https://github.com/TBD54566975/ssi-service/pull/203/files#diff-4fbdd974b372fe74a569df1cb0f988c1429d1b4611b84ef6bc9d17af22923e61R29